src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -45,72 +47,90 @@
                                      :margin-top
                                      10)}])]))
 
-(defn- user-account
-  [{:keys [state name balance percentage-value loading? amount customization-color type emoji metrics?
-           theme on-press]}]
-  (let [watch-only?        (= :watch-only type)
-        empty-type?        (= :empty type)
-        account-amount     (if (= :empty state) "€0.00" amount)
-        account-name       (if (= :empty state) (i18n/label :t/Account 1) name)
-        account-percentage (if (= :empty state) "€0.00" percentage-value)]
-    (if loading?
-      [loading-view
-       {:customization-color customization-color
-        :type                type
-        :theme               theme
-        :metrics?            metrics?}]
-      [rn/pressable
-       {:style    (style/card customization-color watch-only? metrics? theme)
-        :on-press on-press}
-       [rn/view {:style style/profile-container}
-        [rn/view {:style {:padding-bottom 2 :margin-right 2}}
-         [text/text {:style style/emoji} emoji]]
-        [rn/view {:style style/watch-only-container}
-         [text/text
-          {:size   :paragraph-2
-           :weight :medium
-           :style  (style/account-name watch-only? theme)}
-          account-name]
-         (when watch-only? [icon/icon :reveal {:color colors/neutral-50 :size 12}])]]
-       [text/text
-        {:size   :heading-2
-         :weight :semi-bold
-         :style  (style/account-value watch-only? theme)}
-        balance]
-       (when metrics?
-         [rn/view {:style style/metrics-container}
-          [text/text
-           {:weight              :semi-bold
-            :size                :paragraph-2
-            :accessibility-label :metrics
-            :style               (style/metrics watch-only? theme)}
-           account-percentage]
-          (when (not empty-type?)
-            [:<>
-             [rn/view (style/separator watch-only? theme)]
+(def user-account
+  (let [pressed? (reagent/atom false)]
+    (fn [{:keys [state name balance percentage-value loading? amount customization-color type emoji
+                 metrics?
+                 theme on-press]}]
+      (let [watch-only?        (= :watch-only type)
+            empty-type?        (= :empty type)
+            account-amount     (if (= :empty state) "€0.00" amount)
```

Comment:
I think these values will be updated in a separate pr anyway, we'll be soon to start interacting with the backend and will handle these properly then 👍

Reply:
Probably the default value will just get passed a prop

Reply:
sure, probably it's best to remove this value then @mmilad75 and instead add a default prop in place of "€0.00". 
i.e
```
(fn [{:keys [state name ... empty-amount]}]
(let [watch-only?        (= :watch-only type)
            empty-type?        (= :empty type)
            account-amount     (if (= :empty state) empty-amount amount) 
```
or we just use `amount` prop but handle this externally.

We could of course handle this with translation strings as @ilmotta is saying too. So all suggestions welcome. It's just that language != currency selected so that's why I would think better to pull this from the backend 👍

Reply:
I think the approach you suggested with a default prop is even better @J-Son89 👍🏼 The way I see it is that it's better to have a component production ready when we merge it into `develop`, so that we keep `develop` more or less releasable at all times.

Reply:
> I think we don't need `empty-amount` or anything related to it. We already have `:type :empty` which we can say that it's empty. We simply can show the `:balance`.

If that's aligned with designs, sounds good :)

Reply:
Done